src/player.cpp

```diff
@@ -3437,6 +3437,16 @@ void player::on_hit( Creature *source, body_part bp_hit,
             source->add_effect( effect_blind, 2_turns );
         }
     }
+    if( worn_with_flag( "REQUIRES_BALANCE") && !has_effect( effect_downed ) ) {
```

Comment:
`if( one_in( std::max( 2, get_skill_level( skill_dodge ) ) ) ) {` something like this perhaps?